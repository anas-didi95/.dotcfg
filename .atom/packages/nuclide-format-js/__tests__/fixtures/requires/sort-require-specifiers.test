

const {C, B, aB, aA} = require('Z');
const {s, Q, P} = require('X');

aA(aB(B, C, D))
s(Q, P)
