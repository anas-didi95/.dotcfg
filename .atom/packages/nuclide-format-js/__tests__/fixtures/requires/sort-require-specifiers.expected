const D = require('D');

const {s, P, Q} = require('X');
const {aA, aB, B, C} = require('Z');

aA(aB(B, C, D))
s(Q, P)
